chanzuckerberg/idseq-web
# [MapPlayground] Switch tooltips to a new MapTooltip component + other enhancements

### Description
- Standardize on onMouseEnter/onMouseLeave
- Standardize on lat/lng instead of lat/lon
- Update the MapTiler style ID to a new one created by Design
- Design said we won't have a foreseeable use for persistent on-click popups so I took them out of MapPlayground for now.
- Remove radius from the CircleMarker per Design
- Change the tooltips to a new MapTooltip component with an updated style. Make the tooltip feel "naturally sticky" so that it won't close if you hover over the tooltip instead of the marker, and so that it has a slight delay before closing.
- TODO later: Have a circle radius only on selection, adding linking to the tooltip title, etc.

### Demo
![Screen Shot 2019-04-25 at 2 01 10 PM](https://user-images.githubusercontent.com/5652739/56768358-02541980-6763-11e9-992f-32ee7ebb0d0b.png)
![Apr-25-2019 14-03-08](https://user-images.githubusercontent.com/5652739/56768372-08e29100-6763-11e9-9dd8-4be63bd74c6f.gif)

### Test and Reproduce
- Pull the branch, retrieve MAPTILER_API_KEY from Parameter Store, start locally with `MAPTILER_API_KEY=key_here docker-compose up web`
- Visit http://localhost:3000/locations/map_playground
- Hover over bubble markers and look at tooltip style. Move around bubbles and observe that it won't close while over a tooltip but will after you move away.
